Detectors/ITSMFT/ITS/tracking/include/ITStracking/StandaloneDebugger.h

```diff
@@ -15,21 +15,91 @@
 #ifndef O2_ITS_STANDALONE_DEBUGGER_H_
 #define O2_ITS_STANDALONE_DEBUGGER_H_
 
+#include <string>
+#include <iterator>
+
+// Tracker
+#include "DataFormatsITS/TrackITS.h"
+#include "ITStracking/ROframe.h"
+
 namespace o2
 {
 
-class MCCompLabel;
+// class MCCompLabel;
 
 namespace utils
 {
 class TreeStreamRedirector;
 }
 
+// namespace its
+// {
+// class TrackITSExt;
+// }
+
 namespace its
 {
 class Tracklet;
 class Line;
 class ROframe;
+class ClusterLines;
+
+using constants::its::UnusedIndex;
+
+struct FakeTrackInfo {
+ public:
+  FakeTrackInfo();
+  FakeTrackInfo(const ROframe& event, TrackITSExt& track) : isFake{false}, isAmbiguousId{false}, mainLabel{UnusedIndex, UnusedIndex, UnusedIndex, false}
+  {
+    occurrences.clear();
+    for (size_t iCluster{0}; iCluster < 7; ++iCluster) {
+      int extIndex = track.getClusterIndex(iCluster);
+      o2::MCCompLabel mcLabel = event.getClusterLabels(iCluster, extIndex);
+      bool found = false;
+
+      for (size_t iOcc{0}; iOcc < occurrences.size(); ++iOcc) {
+        std::pair<o2::MCCompLabel, int>& occurrence = occurrences[iOcc];
+        if (mcLabel == occurrence.first) {
+          ++occurrence.second;
+          found = true;
+        }
+      }
+      if (!found) {
+        occurrences.emplace_back(mcLabel, 1);
+      }
+    }
+    if (occurrences.size() > 1) {
+      isFake = true;
+    }
+    std::sort(std::begin(occurrences), std::end(occurrences), [](auto e1, auto e2) {
+      return e1.second > e2.second;
+    });
+    mainLabel = occurrences[0].first;
+
+    for (auto iOcc{1}; iOcc < occurrences.size(); ++iOcc) {
+      if (occurrences[iOcc].second == occurrences[0].second) {
+        isAmbiguousId = true;
+        break;
+      }
+    }
+    for (auto iCluster{0}; iCluster < TrackITSExt::MaxClusters; ++iCluster) {
+      int extIndex = track.getClusterIndex(iCluster);
+      o2::MCCompLabel lbl = event.getClusterLabels(iCluster, extIndex);
+      if (lbl == mainLabel && occurrences[0].second > 1 && !lbl.isNoise()) { // if 7 fake clusters -> occurrences[0].second = 1
+        clusStatuses[iCluster] = 1;
+      } else {
+        clusStatuses[iCluster] = 0;
+        ++nFakeClusters;
+      }
+    }
+  }
+  std::vector<std::pair<MCCompLabel, int>> occurrences;
+  MCCompLabel mainLabel;
+  std::array<int, 7> clusStatuses = {-1, -1, -1, -1, -1, -1, -1};
+  bool isFake;
+  bool isAmbiguousId;
+  int nFakeClusters = 0;
+};
```

Comment:
If you want to store this struct using debug streamer, it should be equipped with ``ClassDefNV`` and mentioned in the LinkDef.

Reply:
Uh, thanks, somehow also the `LinkDef` remains a bit obscure to me, but seems reasonable tradeof to be able to stream on file.